diff --git a/dist/index.js b/dist/index.js
index ee5f7e31132e826150cc402ab5ced9cedc384c40..c32ec7fa96982a556a1905e0c873a8845f15fc28 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -301333,7 +301333,10 @@ const startDevServer = async (opts) => {
     });
     const pkg = pathToPkg ? require_(pathToPkg) : {};
     const tsNodePath = require_.resolve('ts-node');
-    const esmLoader = path_1.join(tsNodePath, '..', '..', 'esm.mjs');
+    const esmLoaderAbsolutePath = path_1.join(tsNodePath, '..', '..', 'esm.mjs');
+    const esmLoader = require('path').isAbsolute(esmLoaderAbsolutePath)
+      ? require('url').pathToFileURL(esmLoaderAbsolutePath).href
+      : esmLoaderAbsolutePath;
     const cjsLoader = path_1.join(tsNodePath, '..', '..', 'register', 'index.js');
     const isTypescript = ['.ts', '.tsx', '.mts', '.cts'].includes(ext);
     const maybeTranspile = isTypescript || !['.cjs', '.mjs'].includes(ext);